src/kayenta/report/detail/multipleResultsTable.tsx

```diff
@@ -2,6 +2,7 @@ import * as React from 'react';
 import { chain } from 'lodash';
 import { connect, Dispatch } from 'react-redux';
 import * as classNames from 'classnames';
+import { BreakString } from '@spinnaker/core';
```

Comment:
Is `BreakString` breaking the string based on a fixed length?

Reply:
Not a fixed length, but a fixed width. Normally, the browser will break a string of text at traditional word break points (it's complicated). The `<BreakString>` component just wraps the text in some CSS properties that allow it to break as soon as it reaches the boundary of its box, even if it's in the middle of a word.